<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Desk215" failures="1" tests="4" name="Default test" time="55.381" errors="0" timestamp="2024-07-10T17:20:18 IST">
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="12.56"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="17.256">
    <failure type="java.lang.AssertionError" message="user is navigated to  expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: user is navigated to  expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at StepDefinition.LoginSteps.user_is_navigated_to_the_home_page(LoginSteps.java:54)
at âœ½.user is navigated to the home page(file:///C:/Users/mohamedikfan.m/Desktop/SELENIUM/CucumberProject/src/test/resources/Features/login.feature:11)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="9.868"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="14.546"/>
</testsuite> <!-- Default test -->
